Constants/Maps.ahk.bak

```diff
@@ -1,5 +1,3 @@
-#Include %A_ScriptDir%/Functions/Mouse.ahk
-
```

Comment:
Mouse.ahk is a lib made by @fnzr which helps to zoom out and move screen vertically or horizontally.  It works way better when old A macro for screen movement.